exercises/bob/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
```

Comment:
I wonder if the "He's not very smart" part adds anything. It feels somewhat judgemental?

Reply:
Agreed.

```suggestion
He likes to think that he's very cool.
```